src/azure-cli-core/azure/cli/core/auth/credential_adaptor.py

```diff
@@ -31,6 +30,7 @@ def __init__(self, credential, resource=None, auxiliary_credentials=None):
         self._resource = resource
 
     def _get_token(self, scopes=None, **kwargs):
+        import azure.cli.core.vendored_sdks.winrequests as requests
```

Comment:
Will `winrequests` ever raise `SSLError`?

Reply:
Maybe can we leave `SSLError` to `azure.cli.core.util.handle_exception` to handle? Something like

```py
try:
    raise MyException
except Exception as ex:
    try:
        from requests.exceptions import SSLError
        if isinstance(ex, SSLError):
            print("SSLError captured")
    except ImportError:
        pass
    raise
```

Reply:
No. Let's find an elegant way to handle `ImportError`.

Reply:
The above solution adds more indent in the Exception handling code and introduces broad exception.

I prefer your second idea, which only change the import code.
``` python
try:
    from requests.exceptions import SSLError
except ImportError:
    SSError = type('SSLError', (Exception,), {})
```